collectors/iscsi.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python2
+
+# requires python-rtslib_fb for LIO interaction
```

Comment:
We should make sure we add this package (and any distro-specific aliases) [here](https://github.com/ceph/cephmetrics/blob/master/ansible/roles/ceph-collectd/defaults/main.yml#L6-L25)